ui/objects/semantic-ui/js/objects/search/components/ResultsListItem.jsx

```diff
@@ -1,12 +1,20 @@
-import React, { useContext, useEffect } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import PropTypes from "prop-types";
 import Overridable from "react-overridable";
 import _get from "lodash/get";
 
-import { Grid, Item, Button } from "semantic-ui-react";
+import {
+  Grid,
+  Item,
+  Button,
+  ItemContent,
+  ItemDescription,
+  ItemExtra,
+  Label,
+} from "semantic-ui-react";
 import { withState, buildUID } from "react-searchkit";
 import { SearchConfigurationContext } from "@js/invenio_search_ui/components";
-import {ImageWithFallback} from "./imgFallback"
+import { FeaturedImage } from "./FeaturedImage";
 
 const ItemHeader = ({ title, searchUrl, selfLink }) => {
   const [smallScreen, setSmallScreen] = React.useState(
```

Comment:
Very complicated way of doing display:{none|something} on some breakpoint in less

Reply:
This way it cannot be accepted to main, competes against SUI responsive classes which should be used for this case

Reply:
still an issue
<img width="490" alt="Screenshot 2024-03-26 at 13 55 11" src="https://github.com/VSCHT/restaurovani/assets/646743/cb8f81a5-70fb-48cd-aa38-960207975e9a">